src/ACEtk/interpretation/ContinuousEnergyNeutron/Builder/EnergyDistribution/KalbachMann/src/ACEify.hpp

```diff
@@ -0,0 +1,38 @@
+void ACEify( Table::Data& tData, int jxsRelative ){
+  decltype( auto ) xss = tData.XSS();
+
+  if( tabulated.parameters ){
+    decltype( auto ) p = tabulated.parameters.value();
+    xss 
+      |= ranges::action::push_back( 
+          ranges::view::concat(
+            ranges::view::single( p.first.size() ), p.first, p.second
```

Comment:
Maybe not use concat and ranges::view::single

Reply:
Removed this in favor of a more generic version (i.e., `Tabulated::ACEifywithLOC`).